packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -250,6 +251,80 @@ const CFDsListing = () => {
             ) : (
                 <PlatformLoader />
             )}
+
+            {!is_eu_user && (
+                <div className='cfd-full-row'>
+                    <hr className='divider' />
+                </div>
+            )}
+
+            {!is_eu_user && !is_real && (
+                <div className='cfd-full-row' style={{ paddingTop: '2rem' }}>
+                    <Text weight='bold'>{localize('Deriv cTrader')}</Text>
+                </div>
+            )}
+
+            {is_landing_company_loaded && !is_real
```

Comment:
@hirad-deriv since we need to check all parts based on is_real , it's better to refactor the code and check it once at the top
```suggestion
            !is_real ? () : ()
```

Reply:
This check is temporary and will be removed one week after the release for the real cTrader account.